src/db/index.ts

```diff
@@ -175,3 +179,23 @@ export async function adjustDownvotesForProject (type: 'add' | 'remove', id: Dis
     return { success: true, wasRejected: hasEnoughDownvotes, wasPaused: project.paused, reason: '', project }
   }
 }
+
+export function isEligibleForLicenseCheck (submission: ProjectSubmission): boolean {
+  return !!submission.links.source.includes('github.com')
```

Comment:
Unnecessary boolean coercion
```suggestion
  return submission.links.source.includes('github.com')
```